addons/l10n_jo_edi/models/account_edi_xml_ubl_21_jo.py

```diff
@@ -0,0 +1,302 @@
+from lxml import etree
+
+from odoo import models
+from odoo.exceptions import ValidationError
+from odoo.tools import float_repr
+from odoo.tools.float_utils import float_round
+
+
+JO_CURRENCY = type('JoCurrency', (), {'name': 'JO'})()
```

Comment:
@smetl It's actually needed, and I added a comment explaining why in the v18.0 PR. The comment is as follows:
```python
# There is a need for this dummy currency because:
# 1. In `ubl_20_templates.xml`, the currency name is read from currency like `vals['currency'].name`
# 2. In Jordanian EDI XML documentation, certain locations expect the currency name to be `JO` not `JOD`.
```
If you have a better idea in mind to solve this issue (without breaking all the modules the depend on the ubl base module) plz let me know